04_environment/index.tex

```diff
@@ -1 +1,286 @@
+    \caption{Disaster Epicenter effects}
+    \label{fig:Disaster eye of the storm severity}
+\end{figure}
+
+\subsection{Future work}
+
+There are multiple future work ideas that would be developed in the design aspect such as; a deterministic disaster which would be  designed as a straight line that accumulates during the days, once the threshold has been met then the disaster would occur. Another deterministic disaster idea would be to link the disaster’s magnitude to time. Thus, agents will have to learn from past disaster occurrences, creating a memory. Agents will start learning from past disasters and will be able to forecast the next one. Therefore, the agents will be able to forecast that if a disaster did not surface for a long period of time, then the magnitude of the next disaster will be much larger than the previous one.
+
+Another idea is that agents can invest into forecasting. Therefore, agents will have access to a history of past disasters, which would be utilised to learn and gain knowledge in order to predict the upcoming disasters.
+
+Thus, the more resources invested by the agents in forecasting, the more disaster history data will be provided. This would enable the agents, to apply machine learning techniques to such data, allowing them to acquire more accurate and precise predictions of future events and develop better risk assessments. Therefore, agents will acquire knowledge, when investing in any of their resources.
+
+\section{Common Pool}
+\subsection{Common Pool Background}
+
+A game has been designed where six groups of islands (agents) have access to a common-pool resource. In micro-level, each island intends to maximise its utility while in macro-level, all the islands want to maintain sustainability and be protected by the upcoming disaster. The above description specifies a collective action problem where n-agents (six in our case) are seeking access to a common pool resource that is sufficient to satisfy some agents but not to satisfy all of them.
+
+A Linear Public Goods (LPG) Game has been implemented, where all the agents individually own some resources and try to maintain and increase them, by foraging, but also contribute a part of them to the common pool. This common pool is used for the payment of expenses of the institutional roles and for the protection towards the disaster.
+
+In more detail, the common pool is represented by a structure with following characteristics:
+
+\begin{itemize}
+    \item Common Pool Threshold.
+    \item Amount of Resources.
+\end{itemize}
+
+According to the specifications of a Linear Public Goods Game, the agents should be able to perform the following actions:  
+
+\begin{itemize}
+    \item Request Resource from the Common Pool.
+    \item Contribute Resource to the Common Pool. 
+\end{itemize}
+
+And the Common Pool is responsible for two more actions: 
+
+\begin{itemize}
+    \item Mitigate Disaster.
+    \item Deplete islands after a disaster.
```

Comment:
I'm not sure this is correct - @NGovani I might just be tired at this point but the common pool s not responsible for "depleting" islands of their resources. (??)

Reply:
grammar nazi here:
**Mitigate disaster.**

no capitalization :( plzzz